### PR TITLE
[mobile] switch to png icons

### PR DESCRIPTION
**Pull request by rasom**
status: ready

### CODE REVIEW

**Review by mandrigin — APPROVED** [review]

File: src/status_im/ui/components/icons/vector_icons.cljs [hunk 1]
```diff
@@ -258,14 +158,16 @@
...
   ^{:key name}
-  [react/animated-view {:style               container-style
-                        :accessibility-label accessibility-label}
-   (if-let [icon-fn (get icons name)]
-     (let [icon-vec (icon-fn (match-color color))]
-       (if width
-         (update icon-vec 1 assoc :width width :height height)
-         icon-vec))
-     (throw (js/Error. (str "Unknown icon: " name))))])
+  [react/view
```
Comment (flexsurfer) [inline src/status_im/ui/components/icons/vector_icons.cljs:161]: interesting why was it animated?

**Review by flexsurfer — APPROVED** [review]
what about removing svgs ?

**Comment by rasom** [comment]
@flexsurfer they are used on desktop; right now I'm trying to verify if there is actually any improvement

File: src/status_im/ui/components/icons/vector_icons.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/components/icons/vector_icons.cljs:161]: no idea

**Comment by rasom** [comment]
Teststed on slow device:
1. "Add chat" animation
    I measured time between clicking "add" button, start of animation and end of animation. The first frame is the one where there is no touch icon on "add" button. 30fps, samsung a500d
    

|| pr release build |`develop` release build |
|---| -------- | -------- |
|1.| 11(1)     | 14(9) **this one was really bad**    | 
|2.| 11(1)     | 11(4)     | 
|3.| 11(1)     | 12(5)     | 
|avg| 11(1)     | 12.3(6)     | 

So with png the animation starts much earlier, with svg it is delayed for at least 3 frames (6 at 60fps), which is 100ms. 

2. Wallet -> Send navigation. Measured time between click, start of transition, end of transition
 
|| pr release build |`develop` release build |
|---| -------- | -------- |
|1.| 17(10)     | 18(12)     | 
|2.| 15(9)     | 20(14)     | 
|3.| 16(9)     | 15(10) **allmost the same in this case**    | 
|avg| 16(9.3)     | 17.7(12)     | 

Again, it is close to 100ms difference. Most likely it will be similar number for each transition, as we usually have 4-5 icons on each screen. There is no big difference in the whole transition time, but the animation is delayed in case if svg is used.

**Comment by rasom** [comment]
@errorists 
I think there is no app's logo in that pngs archive https://github.com/status-im/status-react/pull/8195#issuecomment-493945762

**Comment by errorists** [comment]
@rasom gotcha. Since these are not resolution independent anymore, can you tell me at which size (1x) we need the app logo? Is the largest one on the splash screen?

**Comment by rasom** [comment]
@errorists it is 46x46 there https://github.com/status-im/status-react/blob/496f3f1cc78703c3d629f9f458a0e966b04e5c61/src/status_im/ui/screens/intro/styles.cljs#L16

**Comment by errorists** [comment]
thanks, exported one at 46x46 here [Dropbox](https://www.dropbox.com/sh/bdvldm1awqbmd85/AACD_S6_G-wNAC-N7oF0XRtIa?dl=0)

**Comment by rasom** [comment]
@errorists 
also 20x20
https://github.com/status-im/status-react/blob/b3df9d495258d22f3234c7d781fc3350d6c19087/src/status_im/ui/screens/profile/tribute_to_talk/views.cljs#L206

12x12
https://github.com/status-im/status-react/blob/b3df9d495258d22f3234c7d781fc3350d6c19087/src/status_im/ui/screens/stickers/views.cljs#L36

34x34
https://github.com/status-im/status-react/blob/496f3f1cc78703c3d629f9f458a0e966b04e5c61/src/status_im/ui/screens/accounts/create/styles.cljs#L28

**Comment by errorists** [comment]
@rasom we should also use the 46 in all those instances since it's larger, downsizing is perfectly fine with PNGs.

**Comment by rasom** [comment]
@errorists ok then, thanks

**Comment by rasom** [comment]
@errorists it actually was 46 for "S" icon and 111 for the whole thing...

**Comment by errorists** [comment]
@rasom 🤔 111 is a weird number to have but anyway this will be updated with new onboarding. Updated the resources in the dropbox link above, it's 111 now.

**Comment by rasom** [comment]
@errorists I agree but https://github.com/status-im/status-react/blob/496f3f1cc78703c3d629f9f458a0e966b04e5c61/src/status_im/ui/screens/intro/styles.cljs#L15

maybe @flexsurfer knows why

**Comment by Serhy** [comment]
Several issues after migration to PNG (some of them are iOS specific):

1) Some icons became bigger than in develop (Group/Pubic chat icon on Home screen, 'lock' icon in browser bar):

Screenshot 2019-06-12 at 12 24 19

2) Lost icons on 
a) `Fleets settings` screen 
b) `Home -> Search chat -> Messages`
c) `Group chat -> Group info -> Delete chat` **[iOS only]**
Screenshot 2019-06-12 at 12 32 04

3) Two lost icons on sticker selection pad:
Screenshot 2019-06-12 at 12 41 41

4) The `+` char is not centred on a) Dapp permission on DApps screen and b) sticker selection area:

Screenshot 2019-06-12 at 12 52 51

5) Lost upper-arrow icon on Send Message button **[iOS only]**
Screenshot 2019-06-12 at 13 04 54

6) Lost icons on Log Level screen:
Screenshot 2019-06-12 at 13 22 28


7) Lost icons on outgoing/pending/incoming transaction history and filter history screens **[iOS only]**

Screenshot 2019-06-12 at 13 18 40

**Comment by errorists** [comment]
@rasom hey, this has me wondering, do we need to supply the Android specific image formats so `MDPI-HDPI-XHDPI-XXHDPI-XXXHDPI` or will having only the iOS ones like `@1x @2x @3x` be enough? facebook docs [seems to be convinced](https://facebook.github.io/react-native/docs/images) just having the iOS ones is enough. With regards to the sticker icons, I forgot to export them sorry, they are available here [Dropbox](https://www.dropbox.com/sh/7qprgnyoz8ifqg3/AABGLRJDQHfF3GXLsapqnMKfa?dl=0)

**Comment by rasom** [comment]
@Serhy issues should be fixed now

**Comment by rasom** [comment]
@errorists 
> do we need to supply the Android specific image formats so MDPI-HDPI-XHDPI-XXHDPI-XXXHDPI or will having only the iOS ones like @1x @2x @3x be enough? 

have no idea tbh

**Comment by Serhy** [comment]
@rasom it's good, one last item though (sorry didn't notice on the previous build)
**[iOS only]** Lost icon on 'Add to contacts' of users profile who are not in contact list:
Screenshot 2019-06-13 at 18 05 55

**Comment by rasom** [comment]
@Serhy I just pushed fix

**Comment by Serhy** [comment]
@rasom, perfect! Thanks a lot!

**Pull request merged**